shared/vehicles.lua

```diff
@@ -5388,6 +5388,147 @@ QBShared.Vehicles = {
         ['hash'] = `tiptruck2`,
         ['shop'] = 'truck',
     },
+    ['boor'] = {                         --DLC +set sv_enforceGameBuild 2802
+        ['name'] = 'BOOR',
+        ['brand'] = 'Karin',
+        ['model'] = 'boor',
+        ['price'] = 768000,
+        ['category'] = 'offroad',
+        ["categoryLabel"] = 'Off Road',
+        ['hash'] = `boor`,
+        ['shop'] = 'UNRELEASED',
+    },
+    ['broadway'] = {                         --DLC +set sv_enforceGameBuild 2802
+        ['name'] = 'Broadway',
+        ['brand'] = 'Classique',
+        ['model'] = 'broadway',
+        ['price'] = 555000,
+        ['category'] = 'muscle',
+        ["categoryLabel"] = 'Muscle',
+        ['hash'] = `broadway`,
+        ['shop'] = 'UNRELEASED',
+    },
+    ['entity3'] = {                         --DLC +set sv_enforceGameBuild 2802
+        ['name'] = 'Entity MT',
+        ['brand'] = 'Overflod',
```

Comment:
Could you add the brand name as - Överflöd this just keeps it consistent I've already sent in a PR to fix this for the other that were also spelt like this :)